### ISSUE TITLE
Change user agent

### ISSUE THREAD

**Issue by gabrc52** [snobu/destreamer#204]
Would it be possible to change the user agent to a standard Windows+Chrome user agent?

Otherwise, it might be possible for Office365 administrators to tell I'm using `destreamer`.

**Comment by lukaarma** [comment]
You could change our user agent but ffmpeg would still use theirs so you'd have to change that one also for it to be ""stealth"".
I'm using that because a willing admin could easily see that you are using destreamer from the unusual requests that miss stuff like tracking cookies and whatnot.
I wouldn't stress about this because I can't believe that someone would be there looking for people downloading videos from the platform to reprimand them.

TLDR: don't worry, very very unlikely that someone cares enough about some downloads
